libbeat/processors/actions/docs/truncate_fields.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[truncate-fields]]
+=== Truncate fields
+
+The `truncate_fields` truncates a field to a given size if bigger. If the size of the field is smaller than
+the limit, the field is left as is.
+
+`fields`:: List of `from` and `to` pairs to copy from and to.
+`max_bytes`:: Maximum number of bytes in a field. Mutually exclusive with `max_characters`.
+`max_characters`:: Maximum number of characters in a field. Mutually exclusive with `max_bytes`.
+`fail_on_error`:: (Optional) If set to true, in case of an error the changes to
```

Comment:
What you have here is consistent with how we describe this field elsewhere. I think the description needs to be improved, but I'll do that in a follow-on PR so I can apply the change to the docs for other processors.

Adding needs_edit label to remind me.